src/main/resources/scripts/init-script.gradle

```diff
@@ -0,0 +1,175 @@
+import org.gradle.util.GradleVersion
```

Comment:
This should be in "our" package so it does not clash with something else. Move this to `src/main/resources/hudson/plugins/gradle/injection"